src/components/Form.js

```diff
@@ -258,32 +258,36 @@ const reducer = (draft, action) => {
         <Switch>
 
           <Route exact path="/">
-            <ErrorBoundary>
+            <ErrorBoundary useCard>
               <FormStart form={form} onFormStart={onFormStart} />
             </ErrorBoundary>
           </Route>
 
           <Route exact path="/overzicht">
-            <RequireSession expired={sessionExpired} expiryDate={expiryDate}>
-              <RequireSubmission
-                submission={state.submission}
-                form={form}
-                processingError={state.processingError}
-                onConfirm={onSubmitForm}
-                onLogout={onLogout}
-                component={Summary}
-                onClearProcessingErrors={() => dispatch({type: 'CLEAR_PROCESSING_ERROR'})}
-              />
-            </RequireSession>
+            <ErrorBoundary useCard>
+              <RequireSession expired={sessionExpired} expiryDate={expiryDate}>
+                <RequireSubmission
+                  submission={state.submission}
+                  form={form}
+                  processingError={state.processingError}
+                  onConfirm={onSubmitForm}
+                  onLogout={onLogout}
+                  component={Summary}
+                  onClearProcessingErrors={() => dispatch({type: 'CLEAR_PROCESSING_ERROR'})}
+                />
+              </RequireSession>
+            </ErrorBoundary>
           </Route>
 
           <Route exact path="/bevestiging">
-            <RequireSubmission
-              submission={state.submittedSubmission}
-              statusUrl={state.processingStatusUrl}
-              onFailure={onProcessingFailure}
-              onConfirmed={() => dispatch({type: 'PROCESSING_SUCCEEDED'})}
-              component={SubmissionConfirmation} />
+            <ErrorBoundary useCard>
+              <RequireSubmission
+                submission={state.submittedSubmission}
+                statusUrl={state.processingStatusUrl}
+                onFailure={onProcessingFailure}
+                onConfirmed={() => dispatch({type: 'PROCESSING_SUCCEEDED'})}
+                component={SubmissionConfirmation} />
+              </ErrorBoundary>
           </Route>
 
           <Route exact path="/betaaloverzicht">
```

Comment:
This route now has only `<ErrorBoundary>` without the `useCard` prop. Is it intentional?

Reply:
ehm no, the `useCard` prop is present - though it looks like I messed up the indentation which probably caused the confusion!